.storybook/stories/templates/Login-page.stories.mdx

```diff
@@ -0,0 +1,49 @@
+import { Meta, Canvas, Story } from "@storybook/addon-docs";
+import { Heading1, Heading2, Paragraph } from "/components/Typography/src";
+import Button from "/components/Button/src";
+import Alert from "/components/Alert/src";
+
+import Page from "/components/Page/src";
+import Header from "/components/Header/src";
+import HeaderLogo from "/components/HeaderLogo/src";
+import ResponsiveContent from "/components/ResponsiveContent/src";
+import Breadcrumb from "/components/Breadcrumb/src";
+import { AuthenticationCardGroup } from "/components/CardGroup/src/story-template";
+
+import clsx from "clsx";
+
+import "./story-login.css";
+
+<Meta title="Templates/Login page" />
+
+# Summary page
```

Comment:
Hmmm is het eigenlijk de Summary page of the Login page?

Reply:
Login page, i've changed it.